scripts/valhalla_build_extract

```diff
@@ -66,6 +98,19 @@ parser.add_argument(
     type=str,
     default='{}',
 )
+parser.add_argument(
+    "-e",
+    "--extract-tar",
+    help="If specified, will build an extract from an existing tar file at 'mjolnir.tile_extract' and save it to this specified path.",
+    type=str,
+    default=""
+)
```

Comment:
so if someone specifies e.g. `-e new_tar.tar`, we'd write the result to `./new_tar.tar` instead of what specified in the config. the tar specified in the config is instead taken as input.